templates/devtest/fomantic-modal.tmpl

```diff
@@ -47,16 +48,6 @@
 	</div>
 
 	<div class="modal-buttons"></div>
-	<script type="module">
-		for (const el of $('.ui.modal')) {
-			const $btn = $('<button>').text(`Show ${el.id}`).on('click', () => {
-				$(el).modal({onApprove() {alert('confirmed')}}).modal('show');
-			});
-			$('.modal-buttons').append($btn);
-		}
-	</script>
-	<style>
-		.modal-buttons button { margin: 5px; }
-	</style>
+	<script src="{{AssetUrlPrefix}}/devtest.js?v={{AssetVersion}}"></script>
```

Comment:
I guess here it causes duplicate initialization.

Some global code is executed in devtest.js , then the same global code would be executed in "index.js" again (by the "footer.tmpl"

Reply:
Hmm indeed, I see `bootstrap.js` runs twice.

Reply:
Hmm if I removed the duplicate parts, fomantic modal can not init in devtest for some reason, even thought the fomantic code should already have ran.

<img width="1095" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/0a4d980e-8b86-440d-a6c8-87dace4f4473">

Reply:
Then there might be 2 solutions:

1. Find a clear method to avoid duplicate parts. eg: make the "devtest" code as dynamic import.
2. Tolerate the duplicate parts, write some clear comments about the problem and clarify the potential side effects of it.

Reply:
devtest code should work in conjunction with `index.js`, just need to figure out why the `.modal` fails.

Reply:
I guess they just don't co-work. See my another review comment below https://github.com/go-gitea/gitea/pull/25940#pullrequestreview-1553274315

Reply:
Duplication is removed.